data/brands/amenity/fast_food.json

```
@@ -1438,6 +1438,21 @@
         "takeaway": "yes"
       }
     },
+    {
+      "displayName": "Goli Vada Pav",
+      "id": "golivadapav-aba431",
+      "locationSet": {"include": ["in"]},
+      "tags": {
+        "alt_name": "Goli Vadapav No.1",
```

Should the spacing be like this?
```suggestion
        "alt_name": "Goli Vada Pav No.1",
```